<launch>
 <param name="busy" value="0"/>
 <param name="receiving" value="0"/>
 <param name="CONFIG" value="$(find nao_apps)/launch/pepper.corpus"/>

 <node name="synthetiser" pkg="nao_apps" type="nao_speech.py" output="screen">
    <param name="PORT" value="9559"/>
    <param name="IP" value="192.168.101.69"/>
  </node>

 <node name="speechRecognizer" pkg="nao_apps" type="speechRecognizer.py" output="screen">
    <param name="PORT" value="9559"/>
    <param name="IP" value="192.168.101.69"/>
    
  </node>
  <node name="rpc_server" pkg="nao_apps" type="rpc_server.py" output="screen">
    <param name="SERVER_PORT" value="8000"/>
    <param name="SERVER_IP" value="192.168.101.249"/>
    
  </node>
<node name="dialogManager" pkg="nao_apps" type="dialogManager.py" output="screen">
     <param name="PR2_PORT" value="8005"/>
     <param name="PR2_IP" value="192.168.101.249"/>
</node>
    
</launch>

